<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIPreferences</class>
 <widget class="QDialog" name="UIPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="m_pqhblPreferences" stretch="0,0">
     <item>
      <widget class="QListWidget" name="m_pqlwPageList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File List</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rename</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Interface</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="m_pqswPageStack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_2">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Start Directory</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="m_pqrbStartDirectoryRoot">
              <property name="text">
               <string>Root / Computer</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_pqrbStartDirectoryHome">
              <property name="text">
               <string>Home</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="m_pqhblStartDirCustom">
              <item>
               <widget class="QRadioButton" name="m_pqrbStartDirectoryCustom">
                <property name="text">
                 <string>Custom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="m_pqleStartDirectory">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_pqpbStartDirectorySelect">
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pqcbAutoRefreshEnabled">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Auto-refresh directory listing when contents change</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pqcbOpenFileWhenDblClicked">
           <property name="toolTip">
            <string>Rename operations can fail if the file is open in another application.</string>
           </property>
           <property name="text">
            <string>Open file when double-clicked (rename can fail if file is open)</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="m_pqcbNameChangeColourText">
               <property name="text">
                <string>Colour text of rows whose name will change with colour</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="m_pqcbNameChangeHighlightRow">
               <property name="text">
                <string>Highlight rows whose name will change with colour</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="m_pqpbNameChangeTextColour">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="m_pqpbNameChangeHighlightColour">
               <property name="maximumSize">
                <size>
                 <width>25</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="m_pqcbUseCustomLineSpacing">
             <property name="text">
              <string>Use line spacing of </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="m_pqleCustomLineSpacingPxl">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>pixels</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="m_pqcbUseAlternativeFont">
             <property name="text">
              <string>Use alternative font</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_pqpbFontSelectButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="m_pqcbShowConfirmBeforeRename">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Show confirmation message before rename</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pqcbDeactivateSettingsAfterRename">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Deactivate rename settings after rename</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pqcbReSortFileListAfterRename">
           <property name="text">
            <string>Re-sort file list after rename</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="m_pqsiInvCharTableSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Enter a character or string to replace invalid characters</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTableWidget" name="m_pqtwInvalidCharacterTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>0</number>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderCascadingSectionResizes">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderMinimumSectionSize">
               <number>0</number>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Invalid Character</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Replacement</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="m_pqhblCharSubButtonLayout">
              <item>
               <widget class="QLabel" name="m_pqlblInvalidCharTableExplanation">
                <property name="text">
                 <string>&lt;a href=&quot;https://www.inviska.com/rename/instructions/index.shtml#InvalidCharSub&quot;&gt;Explanation Here&lt;/a&gt;</string>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="m_pqpbCharSubRestoreDefaults">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Restore Defaults</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="m_pqpbCharSubSetDefaultsWin">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>&amp;Set Windows Defaults</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QCheckBox" name="m_pqcbShowSideBar">
           <property name="text">
            <string>Show side-bar (Contains buttons for re-ordering files)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UIPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UIPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqlwPageList</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>m_pqswPageStack</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbUseCustomLineSpacing</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleCustomLineSpacingPxl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbUseCustomLineSpacing</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleCustomLineSpacingPxl</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqcbUseAlternativeFont</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqpbFontSelectButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbStartDirectoryCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleStartDirectory</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqrbStartDirectoryCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_pqleStartDirectory</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
